lib/shared/pe_status_check.rb

```diff
@@ -5,6 +5,8 @@
 
 # PEStatusCheck - Shared code for pe_status_check facts
 module PEStatusCheck
+  PUP_PATHS = { server_data: '/opt/puppetlabs/server/data' }.freeze
```

Comment:
server_data is defined here in the array pup_paths, server data is never then consumed

there is also an issue with the array declaration in testing:

```
   /etc/puppetlabs/code/environments/production/modules/pe_status_check/lib/shared/pe_status_check.rb:8: warning: already initialized constant PEStatusCheck::PUP_PATHS
       /opt/puppetlabs/puppet/cache/lib/shared/pe_status_check.rb:8: warning: previous definition of PUP_PATHS was here
```

It may be better to perhaps not use an array for an item that is not consumed